ui/src/publication/log/publication-log.tsx

```diff
@@ -17,16 +17,31 @@ import { FieldLayout } from 'vayla-design-lib/field-layout/field-layout';
 import { PublicationTableItem } from 'publication/publication-model';
 import { Page } from 'api/api-fetch';
 import { PrivilegeRequired } from 'user/privilege-required';
+import { createDelegates } from 'store/store-utils';
+import { trackLayoutActionCreators } from 'track-layout/track-layout-slice';
+import { useTrackLayoutAppSelector } from 'store/hooks';
+import { useAppNavigate } from 'common/navigate';
 
-export type PublicationLogProps = {
-    onClose: () => void;
-};
-
-const PublicationLog: React.FC<PublicationLogProps> = ({ onClose }) => {
+const PublicationLog: React.FC = () => {
     const { t } = useTranslation();
+    const navigate = useAppNavigate();
+
+    const selectedPublicationSearch = useTrackLayoutAppSelector(
+        (state) => state.selection.publicationSearch,
+    );
 
-    const [startDate, setStartDate] = React.useState<Date | undefined>(subMonths(currentDay, 1));
-    const [endDate, setEndDate] = React.useState<Date | undefined>(currentDay);
+    const initialStartDate = selectedPublicationSearch?.startDate;
+    const initialEndDate = selectedPublicationSearch?.endDate;
```

Comment:
Alustusarvoja haetaan nyt reduxista, aiemmin nämä asetettiin aina defaultteihin.

Reply:
Lienee ainakin aiempaa parempi että nämä tallennetaan pysyvästi reduxiin, mutta pitäisiköhän tässä sitten indikoida jotenkin selkeästi että operaattori on valinnut jotkut päivämäärät? Mietin siis pääasiassa keissiä jossa operaattori käy katsomassa jotain vanhaa tietoa julkaisulokista, tekee sen jälkeen viikon-pari töitä muualla, ja navigoi sitten julkaisulokiin. Tällöinhän julkaisulokissa näytetään taas kauan aiemmin valittu päivämääräväli. Onko hyvä että nämä valinnat säilytetään noinkin kauan?